src/main/app/README.md

```diff
@@ -2,25 +2,29 @@
 
```

Comment:
I would change the title from `App` to e.g. `Zaakafhandelcomponent frontend application` ?

Reply:
Adjusted